src/test/java/kg/jarkyn/cobspecserver/doubles/ResponseControllerDouble.java

```diff
@@ -0,0 +1,30 @@
+package kg.jarkyn.cobspecserver.doubles;
+
+import kg.jarkyn.cobspecserver.*;
+
+import java.io.IOException;
+
+public class ResponseControllerDouble extends ResponseController {
+
+    private int timesResponded = 0;
+    private String response;
+
+    public ResponseControllerDouble(String response) {
+        super(null, null);
+        this.response = response;
+    }
+
+    @Override
+    public void respond(Client client) {
+        try {
+            timesResponded++;
+            client.getOutputStream().write(response.getBytes());
```

Comment:
You're right. It probably shouldn't be `Client`'s responsibility.
Yeah, if `StreamHandler` has been introduced for that very reason, use it here. :+1: